lib/observer/src/observer_sys_wx.erl

```diff
@@ -83,14 +92,39 @@ create_sys_menu(Parent) ->
 update_syspage(#sys_wx_state{node = undefined}) -> ignore;
 update_syspage(#sys_wx_state{node = Node, fields=Fields, sizer=Sizer}) ->
     SysInfo = observer_wx:try_rpc(Node, observer_backend, sys_info, []),
-    {Sys, Mem, Cpu, Stats} = info_fields(),
+    {Sys, Mem, Cpu, Stats, Limits} = info_fields(),
     observer_lib:update_info(Fields,
 			     observer_lib:fill_info(Sys, SysInfo) ++
 				 observer_lib:fill_info(Mem, SysInfo) ++
 				 observer_lib:fill_info(Cpu, SysInfo) ++
-				 observer_lib:fill_info(Stats, SysInfo)),
+				 observer_lib:fill_info(Stats, SysInfo)++
+				 observer_lib:fill_info(Limits, SysInfo)),
+
     wxSizer:layout(Sizer).
 
+
+maybe_convert(V) when (is_atom(V) andalso V =:= undefined) -> "Not available";
+maybe_convert(V) -> observer_lib:to_str(V).
+
+get_dist_buf_busy_limit_info() ->
+    fun(Data) ->
+            maybe_convert(proplists:get_value(dist_buf_busy_limit, Data))
+    end.
+
+get_limit_count_info(Count, Limit) ->
+    fun(Data) ->
+            C = proplists:get_value(Count, Data),
+            L = proplists:get_value(Limit, Data),
+            lists:flatten(
+              io_lib:format("~s / ~s ~s",
+                            [maybe_convert(C), maybe_convert(L),
+                             if
+                                 {C, L} =:= {undefined, undefined} -> "";
```

Comment:
Replace with (otherwise C/L will crash if one of them is undefined)
C =:= undefined -> "";
L =:= undefined -> "";